### PR TITLE
added Quadratic Weighted Kappa to the metrics list

### PR DESCRIPTION
**Pull request by AakashKumarNain**
Quadratic Weighed Kappa is a metric that is heavily used to compare the agreement between two raters. For example, it is currently the evaluation metric for one of the [competitions](https://www.kaggle.com/c/petfinder-adoption-prediction#evaluation) on `Kaggle`. There was no native implementation in Keras so I  implemented one for the same.

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 74 days with no activity and the `awaiting review` label has been applied.

**Comment by pavithrasv** [comment]
I think we should add this to `add-ons` for now and and merge into core based on how widely it is used. https://github.com/tensorflow/addons/tree/master/tensorflow_addons

**Comment by AakashKumarNain** [comment]
Sure. When I submitted this PR, `addons` wasn't there but yeah it makes sense to add it to `addons` first. I would open a PR there then. Thanks. Closing this one for now
